### ISSUE TITLE
Zmanim

### ISSUE THREAD

**Issue by lukecarbis** [hebcal/hebcal-swift#2]
Is there any chance that Zmanim will make it into the Swift version, same as JS?

**Comment by mjradwin** [comment]
Thanks for asking! Yes, there's a chance, but to be honest we don't have any planned timeline. This could be a year or more in the future. We developed this port because we were working on an Apple Watch app for Hebcal. We want to eventually add candle-lighting times to that app, which will require integrating a Swift solar calculation library.

In our research, we stumbled across https://github.com/ceeK/Solar which seems like it might be a good Swift library for sunrise/sunset engine. Those calculations do the majority of the work for Zmanim; the remainder can be ported from our JS code (which in turn was largely ported from the excellent https://github.com/KosherJava/zmanim by Eliyahu Hershfeld)
